Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/SwitchTeamMessage.java

```diff
@@ -15,20 +17,39 @@
     private final int playerId;
     private final TeamColour team;
 
+    /**
+     * Intializes the SwitchTeamMessage class.
+     *
+     * @param playerId The id of the player, not null.
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)